Corrected instance counts for multiple profiles

ManojNair opened the pull request:
Corrected the instances count for the section Considering multiple profiles. It says minimum as 2 while the image shows 3 and maximum has 10 while the image shows 12.

azurecla commented:
Hi __@ShirleyWang-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shwang). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ManojNair committed:
Corrected instance counts for multiple profiles

Corrected the instances count for the section Considering multiple profiles. It says minimum as 2 while the image shows 3 and maximum has 10 while the image shows 12.
